src/zc/buildout/easy_install.py

```diff
@@ -269,6 +273,8 @@ def __init__(self,
 
         self._index_url = index
         path = (path and path[:] or []) + buildout_and_setuptools_path
+        if force_eggs:
+            path = [p for p in path if not p.endswith('site-packages')]
```

Comment:
I am uneasy with this way of checking `site-packages`. Would anyone have a better solution ?

Reply:
No ideas on solutions, I haven't even looked at the problem, but I'm now wondering about Debian's `dist-packages`.